src/lib/docker-utils.coffee

```diff
@@ -76,6 +75,12 @@ module.exports = class DockerUtils extends DockerToolbelt
 			log("Unsupported delta version: #{version}. Falling back to regular pull")
 			return @fetchImageWithProgress(imgDest, fullDeltaOpts, onProgress)
 
+		# Since the supervisor never calls this function without a source anymore,
+		# this should never happen, but we handle it anyways.
+		if !deltaSource?
+			log("Delta from scratch unsupported. Falling back to regular pull")
```

Comment:
I would just say something like "Performing regular pull due to lack of delta source" or sth. It might scare people seeing unsupported.

Reply:
Good point, will change it